public/app/routes/routes.tsx

```diff
@@ -533,6 +533,18 @@ export function getAppRoutes(): RouteDescriptor[] {
         () => import(/* webpackChunkName: "BookmarksPage"*/ 'app/features/bookmarks/BookmarksPage')
       ),
     },
+    {
+      path: '/admin/provisioning',
+      component: SafeDynamicImport(
+        () => import(/* webpackChunkName: "RepositoryListPage"*/ '../features/provisioning/RepositoryListPage')
+      ),
+    },
+    {
+      path: '/admin/provisioning/new',
+      component: SafeDynamicImport(
+        () => import(/* webpackChunkName: "NewRepositoryPage"*/ '../features/provisioning/NewRepositoryPage')
+      ),
+    },
```

Comment:
Nit: Just for consistency with the other component imports in this file

```suggestion
    {
      path: '/admin/provisioning',
      component: SafeDynamicImport(
        () => import(/* webpackChunkName: "RepositoryListPage"*/ 'app/features/provisioning/RepositoryListPage')
      ),
    },
    {
      path: '/admin/provisioning/new',
      component: SafeDynamicImport(
        () => import(/* webpackChunkName: "NewRepositoryPage"*/ 'app/features/provisioning/NewRepositoryPage')
      ),
    },
```